modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1720,6 +1759,86 @@ GAPI_EXPORTS GMat warpAffine(const GMat& src, const Mat& M, const Size& dsize, i
                              int borderMode = cv::BORDER_CONSTANT, const Scalar& borderValue = Scalar());
 //! @} gapi_transform
 
+/** @brief Parses output of SSD network.
+Extracts detection information (box, confidence, label) from SSD output and
+filters it by given confidence and label.
+
+@note Function textual ID is "org.opencv.core.parseSSDWL"
+
+@param in Input CV_32F tensor with {1,1,N,7} dimensions.
+@param in_sz Size to project detected boxes to (size of the input image).
+@param confidence_threshold Confidence threshold, if confidence of the
+detection is smaller than this threshold, detection is rejected.
+@param filter_label If provided (!= -1), only detections with
+given label will get to the output.
+@return a vector of detected boxes and a vector of appropriate labels.
+*/
+GAPI_EXPORTS core::GDetections parseSSD(const GMat& in,
+                                        const GOpaque<Size>& in_sz,
+                                        const float& confidence_threshold = 0.5f,
+                                        const int&   filter_label = -1);
+
+/** @overload
+Extracts detection information (box, confidence, label) from SSD output and
```

Comment:
I assume this isn't correct. Only box?

Reply:
No. We extract all information from the SSD output.